packages/browsers/src/detectPlatform.ts

```diff
@@ -19,7 +19,9 @@ export function detectBrowserPlatform(): BrowserPlatform | undefined {
         ? BrowserPlatform.MAC_ARM
         : BrowserPlatform.MAC;
     case 'linux':
-      return BrowserPlatform.LINUX;
+      return os.arch() === 'arm64'
```

Comment:
Nit: It would be better to have this in a variable like the platform as we reference it 4 times